lib/pry/commands/ls/constants.rb

```diff
@@ -3,15 +3,16 @@
 class Pry
   class Command::Ls < Pry::ClassCommand
     class Constants < Pry::Command::Ls::Formatter
+      DEPRECATED_CONSTANTS = [:Fixnum, :Bignum, :TimeoutError, :NIL, :FALSE, :TRUE]
```

Comment:
me neither. i hope Ruby implements `Object.deprecated_constant?()` in the future because maintaining the list ourselves isn't ideal, it doesn't cover other constants that might be deprecated in the future, or other Ruby code that uses the `deprecate_constant` feature.